packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5807,6 +5807,7 @@ export default class Meeting extends StatelessWebexPlugin {
 
     // publish the streams
     if (this.mediaProperties.audioStream) {
+      this.setSendNamedMediaGroup(MediaType.AudioMain);
```

Comment:
what about the case when the target language code is changed? In that case we also need to call setSendNamedMediaGroup(), right? but I don't see this in this PR

Reply:
Cantina will call setSendNamedMediaGroup() when changed the target language in UI. I will submit another cantina PR for this later.